go3/tasks.py

```diff
@@ -18,20 +18,20 @@
 from django.conf import settings
 from django.http import HttpResponse
 
-def do_daily_tasks(request):
-    if int(request.GET.get('key',0)) == settings.ROUTINE_TASK_KEY:
-        async_task('gig.tasks.delete_old_trashed_gigs')
-        async_task('gig.tasks.archive_old_gigs')
-        async_task('gig.tasks.send_snooze_reminders')
-        async_task('stats.tasks.collect_band_stats')
-        return HttpResponse('did it!')
-    else:
-        return HttpResponse()
+# def do_daily_tasks(request):
+#     if int(request.GET.get('key',0)) == settings.ROUTINE_TASK_KEY:
+#         async_task('gig.tasks.delete_old_trashed_gigs')
+#         async_task('gig.tasks.archive_old_gigs')
+#         async_task('gig.tasks.send_snooze_reminders')
+#         async_task('stats.tasks.collect_band_stats')
+#         return HttpResponse('did it!')
+#     else:
+#         return HttpResponse()
```

Comment:
Can this code just be deleted?

Reply:
Oops - thought I did. I got rid of the code in urls.py but only commented ot the tasks.py for testing and then didn't delete. d'oh.